pkg/sessions/tests/session_store_tests.go

```diff
@@ -36,6 +39,12 @@ type sessionStoreFunc func() sessionsapi.SessionStore
 // store to fast forward expiry of sessions.
 type PersistentStoreFastForwardFunc func(time.Duration) error
 
+// LookUpMappingFunc is used to check the current mapping from user email to stored session
```

Comment:
Not sure if these are great, but at least consistent with `perisistentFastForward`